docs/pbc/all/dynamic-multistore/202410.0/base-shop/faq.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Dynamic Multistore FAQ
+description: This document contains frequently asked questions for Dynamic Multistore.
+past_updated: Nov 1, 2024
+template: howto-guide-template
+last_updated: Nov 1, 2024
+---
+
+This document contains frequently asked questions about Dynamic Multistore feature.
+
+## Questions
+
+### How to check if Dynamic Multistore is enabled on your environment?
+
+#### Option #1 (On env side)
+- Verify the value of `SPRYKER_DYNAMIC_STORE_MODE` environment variable, if it's exists and it equals to `true` then you project operates in Dynamic Multistore mode.
+#### Option #2 (On interface side)
+- Navigate to
+  http://backoffice.eu.spryker.local/store-gui/list, verify if `Edit` button is available on each store on the page.
+
+### What is the difference of how Storefront and GlueStorefront (as well as Glue) vs Backoffice and Merchant Portal works in Dynamic Multistore mode?
+
+In both cases, the applications works with the several stores withing the one region, but:
```

Comment:
Reference to docs/pbc/all/dynamic-multistore/202410.0/base-shop/difference-between-modes.md ?